apps/docs/content/guides/platform/manage-your-subscription/change-your-subscription-plan.mdx

```diff
@@ -0,0 +1,42 @@
+---
```

Comment:
I'm undecided if I preferred this section as one page, I feel like this might be a bit too scattered and require too much clicking through? Especially "manage your billing details", which is really just an admonition.

Maybe if we put it on one page with a table of contents up top? I can figure out something to auto-generate the table of contents if that would be helpful. (I know we already have the sidebar, so I meant something in the main content area)

Reply:
Yeah, I know what you mean. I'm not happy with the current split either. I'll try the one page approach.

Reply:
It's one page now. Like it way more. Can probably still be improved (e.g. the table of contents you suggested), but would leave it for now. Changes don't go to master directly. We're gonna have a look at all pages again once we have everything.

Reply:
Also stopped using a partial for the billing details section. I think it's better to have different wording for the two occurences.